### PR TITLE
upgrade dependencies version at all

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #130** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/130" target="_blank">Graphite</a> 
    * **PR #131** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/131" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/131?utm_source=stack-comment).

File: subprojects/bounded_context/place/infra/src/main/kotlin/club/staircrusher/place/infra/adapter/out/web/KakaoProperties.kt [hunk 1]
```diff
@@ -1,9 +1,8 @@
 package club.staircrusher.place.infra.adapter.out.web
 
 import org.springframework.boot.context.properties.ConfigurationProperties
-import org.springframework.boot.context.properties.ConstructorBinding
 
 @ConfigurationProperties("scc.kakao")
-data class KakaoProperties @ConstructorBinding constructor(
```
Comment (Zeniuus) [inline subprojects/bounded_context/place/infra/src/main/kotlin/club/staircrusher/place/infra/adapter/out/web/KakaoProperties.kt:7]: 오 이거 없어도 되나요?

**Review by Zeniuus — APPROVED** [review]

File: subprojects/bounded_context/place/infra/src/main/kotlin/club/staircrusher/place/infra/adapter/out/web/KakaoProperties.kt (hunk 1 above)
Comment (bellatoris) [reply subprojects/bounded_context/place/infra/src/main/kotlin/club/staircrusher/place/infra/adapter/out/web/KakaoProperties.kt:7]: 이게 아예 사라졌어요 ㅋㅋ

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/131).

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/131).

**Pull request merged**